Tools/Scripts/webkitpy/tool/commands/upload_unittest.py

```diff
@@ -34,6 +34,7 @@
 from webkitpy.tool.commands.commandtest import CommandsTest
 from webkitpy.tool.commands.upload import *
 from webkitpy.tool.mocktool import MockOptions, MockTool
+from webkitscmpy import mocks as wmocks
```

Comment:
Nit: I wonder if just importing `webkitcorepy.mocks` and `webkitscmpy.mocks` and being explicit throughout would be clearer? But definitely not worth blocking this on!